app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -56,92 +67,151 @@ class Config {
 
   Future<String> get githubOAuthToken => _getSingleValue('GitHubPRToken');
 
-  Future<String> get nonMasterPullRequestMessage =>
-      _getSingleValue('NonMasterPullRequestMessage');
+  String get nonMasterPullRequestMessage =>
+      '''This pull request was opened against a branch other than _master_. Since Flutter pull requests should not normally be opened against branches other than master, I have changed the base to master. If this was intended, you may modify the base back to {{branch}}. See the [Release Process](https://github.com/flutter/flutter/wiki/Release-process) for information about how other branches get updated.
```

Comment:
Nit: this line is too long, consider making it be multiple lines. Also applies to below.

Reply:
Due to the way multilines are handled in Dart, I don't think we can fit these values within a width of 120 characters without making it show up when the GitHub bot posts it. Adding newlines in a multiline string shows in the final string.

I have separated the lines by sentences instead which makes it easier to read, is this okay?

Reply:
1. Logic is more important than format, so I would keep the lines in one place if they are logically one thing.
2.  There's another way to solve it:
```
var s2 = 'You can create ' + 
    'multi-line strings like this one.';
```

Reply:
You can do it a bit more simply as:
```
var s2 = 'foo bloo bar'
  'fizz flazz bazz';
```

Reply:
Thank you both for this! I was able to clean it up and retain the original formatting.